php/PhpJsonApiSamples/Default.html

```diff
@@ -31,41 +31,45 @@
 	<form id="form1" runat="server">
 	<table style="width:40%">
 		<tr><th>Select any option </th></tr>
-		<!--<tr> 
-			<td><a href="ClientSideEncryptionDemo.aspx">ClientSide Encryption </a> </td>
-		</tr>   -->
+		<tr> 
+                        <!-- Demo1 checks the client side validations and then submit the valid form with PayTrace client side library  -->
+			<td><a href="ClientSideEncryptionDemo1.html">Client Side Encryption Demo 1 </a> </td>
+		</tr>   
+                <tr> 
+			 <!-- Demo2 submits the form with PayTrace client side library as soon as form is submitted. -->
+                        <td><a href="ClientSideEncryptionDemo2.html">Client Side Encryption Demo 2 </a> </td>
+		</tr> 
 		<tr>
-                    <td><a href="OAuthTokenJson.php">Authentication Token</a> </td>
+                        <td><a href="OAuthTokenJson.php">Authentication Token</a> </td>
   		</tr>
 		<tr>
 			<td><a href="KeyedSaleJson.php">Keyed Sale (Card-Not Present)</a> </td>
   		</tr>
 		<tr>
 			<td><a href="SwipedSaleJson.php">Swiped Sale (Card Present)</a> </td>
- 		 </tr>
+ 		</tr>
 		<tr>
+                        <td><a href="VaultSaleByCustomerIDJson.php">Vault Sale By Customer ID</a> </td>
+  		</tr>
+                <tr>
 			<td><a href="KeyedAuthorizationJson.php">Keyed Authorization(Card-Not Present)</a> </td>
   		</tr>
 		<tr>
 			<td><a href="CaptureJsonUI.php">Capture Authorized Transaction</a> </td>
   		</tr>
 		<tr>
 			<td><a href="KeyedRefundJson.php">Keyed Refund</a> </td>
- 		 </tr>
+ 		</tr>
 		<tr>
 			<td><a href="VoidTransactionJsonUI.php">Void Transaction</a> </td>
   		</tr>
 		<tr>
-			<td><a href="CreateCustomerJson.php">Create Customer Profile</a> </td>
+			<td><a href="CreateCustomerJson.php">Create Customer Profile (Tokenization)</a> </td>
  		</tr> 
- 		<tr>
-    		<td><a href="VaultSaleByCustomerIDJson.php">Vault Sale By Customer ID</a> </td>
-  		</tr>
+ 		
```

Comment:
The verbiage for these links seem more self explanatory so not sure if you need a description. Just first two client encryption seem a little confusing of what they actually are for. ex demo1 and demo2 doesn't really say what they do.